tasks/dogstatsd.py

```diff
@@ -175,7 +175,7 @@ def integration_tests(ctx, race=False, remote_docker=False, go_mod="mod", timeou
     Run integration tests for dogstatsd
     """
     if sys.platform == 'win32':
-        raise Exit(message='dogstatsd integration tests are not supported on Windows', code=0)
```

Comment:
Is this an intended change? Does not seem to have to do anything with docker images.

Reply:
Yes it is ! Since I had to change the way the tests were run, I was encountering errors.

It's indeed a bit stretched from the PR initial goal but it would be complicated to do it in 2 different PRs without breaking the CI.